orderer/consensus/etcdraft/chain.go

```diff
@@ -434,6 +436,14 @@ func (c *Chain) Halt() {
 	}
 }
 
+// HaltExternal stops the chain but does not call the haltCallback function.
+// This is intended to be used by the channel participation API when removing
+// a channel.
+func (c *Chain) HaltExternal() {
+	c.haltCallback = nil
```

Comment:
Isn't this callback an empty callback in case of a channel participation API?

Reply:
That does appear to currently be the case, yes, but it's my understanding that will change shortly to allow it to switch to a follower/inactive chain as applicable.

Reply:
But I thought the long term goal is to not have any inactive chain? @tock-ibm  ?